### ISSUE TITLE
Add new release repositories

### ISSUE THREAD

**Issue by RoBeau** [ros2-gbp/ros2-gbp-github-org#399]
* Name of the release team: Boeing

* Release repositories to add:

  * [gazebo_set_joint_positions_plugin](https://github.com/Boeing/gazebo_set_joint_positions_plugin)
    * [ros/rosdistro source entry](https://github.com/ros/rosdistro/blob/6bc055ea8f82b320b3234d9341bb537f79463e21/humble/distribution.yaml#L727C1-L735C22)

**Comment by clalancette** [comment]
> * [gazebo_set_joint_positions_plugin](https://github.com/Boeing/gazebo_set_joint_positions_plugin)

Before we can put this one in, the package name needs to change to match the key name.  That is, the package name here should be `boeing_gazebo_set_joint_positions_plugin`.

**Comment by RoBeau** [comment]
Hey Chris, I have updated the package name and submitted a pull request on ros distro to update the source url's.

https://github.com/ros/rosdistro/pull/39454

**Comment by RoBeau** [comment]
Hey Chris, we have decided to drop the boeing on these package. I will close and reopen a new ticket once the rosdistro is updated
